rfc/rfc021-vp_token-grant-type.md

```diff
@@ -167,56 +166,27 @@ The following example shows a request to the Presentation Definition endpoint:
     GET /presentation_definition?scope=a+b HTTP/1.1
     Host: as.example.com
 
-## 6. Scopes
-
-The scope parameter is used to indicate which Verifiable Credentials are required to access a resource.
-The OAuth 2.0 Authorization Framework allows the use of multiple scopes. 
-To simplify processing, this specification only allows a single main scope and multiple sub-scopes.
-Allowing mixing of different main scopes in a single access token request is out of scope of this specification.
-The Authorization Server MUST reject any request that contains multiple main scopes.
-
-The main scope MUST be the first value in the scope parameter.
-It MUST be present and MUST be a string and can't contain `/`.
-A sub-scope is defined as any scope after the main scope.
-It MUST be a string and can contain `/` to indicate specific resources that need to be accessed.
-It SHOULD follow the format `resource/id.operation` where `resource` is the resource type, `id` is the specific identifier of the resource and `operation` is the operation that is allowed on the resource.
-The example below indicates read and write access to a resource with identifier 5:
-
-    resource/5.rw
-
-The following example shows a request to the token endpoint with a main scope and a sub-scope:
-
-    POST /token.oauth2 HTTP/1.1
-    Host: as.example.com
-    Content-Type: application/x-www-form-urlencoded
-
-    grant_type=vp_token-bearer
-    &assertion=eyJhbGciOiJFUzI1NiIsImtpZCI6IjE2In0.
-    eyJpc3Mi[...omitted for brevity...].
-    J9l-ZhwP[...omitted for brevity...]
-    &presentation_submission= {..}
-    &scope=a_scope+a_resource:5:read
-
-## 7. Access Token Introspection
+## 6. Access Token Introspection
 
 The Authorization Server MAY support an access token introspection endpoint as defined in OAuth 2.0 [RFC7662].
 The introspection endpoint SHOULD map the fields as follows:
 
 * `iss`: The issuer of the access token (DID).
-* `sub`: The subject of the access token (DID).
+* `sub`: The subject of the access token (DID), this is the resource owner. Usually the same as the `iss` field.
```

Comment:
this section is not really add anything useful?

Reply:
Entire chapter 6 or specific fields?

Reply:
May be useful after all, since we don't have a separate introspection RFC, and it adds some claims specific for this RFC (regarding the input descriptors)